app/Models/Post.php

```diff
@@ -4,10 +4,19 @@
 
 use Conner\Tagging\Taggable;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
 class Post extends Model
 {
     use Taggable;
+    use SoftDeletes;
```

Comment:
Totally subjective, but you can chain these together if you want: `use Taggable, SoftDeletes;`